### PR TITLE
Add weather alerts

### PR DESCRIPTION
**Pull request by THEKaminsky**
- Add Alerts component - attempted to do this in ES6 was so there wouldn't be conflicts with @aaddaamm doing the ES6 switch over.
- Add Ticker component

@begizi It was pretty good with the storms rolling through tonight. Seemed to work like a dream.

### CODE REVIEW

File: client/src/components/alert/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import moment from "moment";
+
+class Alert extends React.Component{
+  render() {
+    var time = moment.unix(this.props.expires).format("MMM DD ha");
```
Comment (aaddaamm) [inline client/src/components/alert/index.js]: lets make this variable a const instead. it's best for variables that don't change.

Comment (THEKaminsky) [inline client/src/components/alert/index.js]: good call

**Comment by THEKaminsky** [comment]
I also realized that my file types. needed to be switched over to .jsx

File: client/src/components/alert/index.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import moment from "moment";
+
+class Alert extends React.Component{
+  render() {
+    const time = moment.unix(this.props.expires).format("MMM DD ha");
```
Comment (aaddaamm) [inline client/src/components/alert/index.jsx]: It's also a good idea to destructure your prop variables as well like `const {time, expires} = this.props;` this way you don't have to type this.props every time you want to use a variable. granted, this component is small but it's a handy trick.

File: client/src/components/alert/index.jsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+    return (
+      <div className="alert">
+        { this.props.title } <br />Expires: { time }
```
Comment (aaddaamm) [inline client/src/components/alert/index.jsx]: lets remove the whitespace padding on these.

Comment (begizi) [inline client/src/components/alert/index.jsx]: @THEKaminsky What do you think about having the expires in it's own div with classname and we can give it a smaller font size?

Comment (THEKaminsky) [inline client/src/components/alert/index.jsx]: screen shot 2016-07-25 at 10 12 27 am
 this is what we have right now. how much smaller of a font size are you thinking?

Comment (begizi) [inline client/src/components/alert/index.jsx]: Probably whatever you think looks good. Maybe 50-75%ish the text above

File: client/src/components/ticker/index.jsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import React from "react";
```
Comment (begizi) [inline client/src/components/ticker/index.jsx]: Should we pull this out to its own commit? It seems like we are using it before it's getting added. At the very least, it should get moved to the commit where it's used

Comment (begizi) [inline client/src/components/ticker/index.jsx]: Actually. This whole commit can probably just get squashed into the first

File: client/src/components/ticker/index.jsx [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+    this.state = {
+      iteration: 0
+    };
+  };
+
+  componentDidMount(){
+    const { timeOut } = this.props;
+
+    this.interval = setInterval( ()=>{
```
Comment (begizi) [inline client/src/components/ticker/index.jsx]: Small nitpick, but we probably don't need the space before the function. And we can probably use a space around the arrows. ie `setInterval(() => {`

File: client/src/components/ticker/index.jsx [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+  };
+
+  componentWillUnmount(){
+    clearInterval(this.interval);
+  };
+
+  render(){
+    const { children } = this.props;
+
+    return (
```
Comment (begizi) [inline client/src/components/ticker/index.jsx]: Don't need the line break here. Small enough for 1 line

File: client/src/components/weather/index.js [hunk 7]
```diff
@@ -54,15 +54,16 @@ module.exports = React.createClass({
             <ForecastDay
               className="forecast"
               title="Today"
-              data={ this.state.data.daily.data[0] } />
+              dailyData={ this.state.data.daily.data[0]}
```
Comment (begizi) [inline client/src/components/weather/index.js]: Inconsistant padding

File: client/src/components/forecast-day/index.js [hunk 8]
```diff
@@ -11,18 +13,33 @@ module.exports = React.createClass({
...
     }
+  },
+
+  todayAlert: function () {
+    return (
+      <Ticker>
+        {
+          this.props.alertData.map((a, i)=>{
+            return (
+              <Alert key={i} title={ a.title} expires={a.expires} />
```
Comment (begizi) [inline client/src/components/forecast-day/index.js]: Inconsistent padding for `title={}`

File: client/src/components/forecast-day/index.js [hunk 9]
```diff
@@ -24,5 +25,19 @@ module.exports = React.createClass({
...
     }
+  },
+
+  todayAlert: function () {
+    return (
+      <Ticker>
+        {
+          this.props.alertData.map((a, i)=>{
+            return (
+              <Alert key={ i } title={ a.title } expires={ a.expires } />
```
Comment (begizi) [inline client/src/components/forecast-day/index.js]: We're using Alert in this commit before it's defined in the next. All the changes in this file can just get moved to the next commit. The only addition in this commit should be the ticker component files
